CONTRIBUTING.md

```diff
@@ -0,0 +1,29 @@
+# Contributing to AutoBot
+
+Thank you for your interest in AutoBot. It's an ML Marketplace to buy/sell/auction the datasets of your choice to train ML Models.
+
+## Getting Started
+
+ - Check out AutoBot's [open issues](https://github.com/Abhishek-kumar09/AutoBot/issues) here. Contributions can be made by creating Pull Requests after forking the repository.
+ - Another easy way to start contributing is to *file an issue*. Issues can include bug fixes, adding new features, or updating the documentation that looks outdated.
+ 
+ ### Steps To Contribute:
+ 
+- Fork and clone the repository `git clone https://github.com/Abhishek-kumar09/AutoBot.git`
```

Comment:
Better is to provide the link of their forked repo.
Like: `git clone https://github.com/<YOUR_USER_NAME>/AutoBot.git`